lesson9/script/index.js

```diff
@@ -0,0 +1,49 @@
+/*норма*/
+let userName = prompt("привет как тебя зовут?");
+let userAge = prompt("скільки тобі років?");
+let userYearofbirth = prompt("якого року твоє народження?");
+
+let number1 = 1993;
```

Comment:
No need to have this here. Since you request all info from user (userName, userAge, userYearofbirth )